tests/apps/mip/snapshots/snap_test_apps_mip_files.py

```diff
@@ -1,3 +1,4 @@
+# -*- coding: utf-8 -*-
```

Comment:
Do we need this line?

Reply:
No, this is the default in python3, I'll remove it.

Reply:
great